dataworkspace/dataworkspace/apps/applications/views.py

```diff
@@ -425,6 +428,46 @@ def visualisation_users_with_access_html_GET(request, gitlab_project):
     )
 
 
+def visualisation_users_with_access_html_POST(request, gitlab_project):
+    application_template = ApplicationTemplate.objects.get(
+        gitlab_project_id=gitlab_project['id']
+    )
+    user_id = request.POST['user-id']
+    user = get_user_model().objects.get(id=user_id)
+
+    content_type_id = ContentType.objects.get_for_model(user).pk
+
+    with transaction.atomic():
+        try:
+            permission = ApplicationTemplateUserPermission.objects.get(
+                user=user, application_template=application_template
+            )
+        except ApplicationTemplateUserPermission.DoesNotExist:
+            # The permission could have been removed by another request. We
+            # could surface an error, but the state is what the user wanted:
+            # the user does not have access.
+            pass
+        else:
+            permission.delete()
+
+            LogEntry.objects.log_action(
+                user_id=request.user.pk,
+                content_type_id=content_type_id,
+                object_id=user.id,
+                object_repr=force_text(user),
+                action_flag=CHANGE,
+                change_message=f'Removed visualisation {application_template} permission',
+            )
+
+    messages.success(
+        request,
+        f'Removed {user.get_full_name()}\'s view access from {gitlab_project["name"]}',
```

Comment:
```suggestion
        f'{user.get_full_name().capitalize()}\' can no longer view the ‘{gitlab_project["name"]}’ project.',
```

Possible alternative text?

Reply:
Will change (although without capitalise: e.g. since I think would change "Michal Charemza" -> "Michal charemza")